templates/policygen/forseti/tf_based/iam_allow_bindings.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPIAMAllowedBindingsConstraintV1
+metadata:
+  name: whitelist_owner_role
+spec:
+  severity: high
+  match:
+    target:
+    - "{{get . "target"}}"
+  parameters:
+    mode: whitelist
+    role: {{get . "role"}}
+    members:
+    {{- range .bindings}}
```

Comment:
Why is this field called bindings when it's for members?

Reply:
Done